birdhouse/deployment/install-deploy-notebook

```diff
@@ -29,9 +29,10 @@ if [ ! -e "$REPO_ROOT/birdhouse/deployment/trigger-deploy-notebook" ]; then
     exit 2
 fi
 
+. "$REPO_ROOT/birdhouse/env.local"
```

Comment:
Humm, since we can not expect `JUPYTERHUB_USER_DATA_DIR` to always be in `env.local` since it is optional, `install-deploy-notebook` can't use it in `envsubst`.

Reply:
I think it's fine this is not configurable, then we can hardcode it.  All the other bind mount locations are not configurable either.

Oh right, move it to `common.env`, then `./pavics-compose.sh` have access and this `install-deploy-notebook` can source it and use in `envsubst` again.

Reply:
see https://github.com/bird-house/birdhouse-deploy/pull/16/commits/75b7926ac79e366a38c2a038701e2ce55faef1aa

Reply:
Yeah, or old style hardcode like I did.